### PR TITLE
Add ProtonMail

### PR DESCRIPTION
**Pull request by katzrkool**
Added Protonmail

### CODE REVIEW

File: sites.json [hunk 1]
```diff
@@ -4373,7 +4373,17 @@
             "prosper.com"
         ]
     },
-
+
```
Comment (Nightfirecat) [inline sites.json]: Delete spaces.

File: sites.json [hunk 2]
```diff
@@ -4373,7 +4373,17 @@
...
     },
-
+    
+    {
+         "name": "ProtonMail",
+         "url": "https://mail.protonmail.com/account",
+         "difficulty": "easy",
+         "notes": "Log into your account -> Click Settings -> Scroll Down to the Delete My Account Button -> Type why you are leaving, password, and 2 Factor Authentication code if enabled -> Click Delete",
+         "domains": [
+         "protonmail.com"
```
Comment (Nightfirecat) [inline sites.json]: Should be indented one level further.

File: sites.json [hunk 3]
```diff
@@ -4373,7 +4373,17 @@
...
+         ]
+    },
+
```
Comment (Nightfirecat) [inline sites.json]: Delete spaces.

**Comment by neeklamy** [comment]
Thank you @katzrkool for submitting this PR and @Nightfirecat for checking over it.

**Pull request merged**